sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/utils/ErrorLogHelper.java

```diff
@@ -275,6 +287,7 @@ public static synchronized File getNewMinidumpSubfolderWithContextData(Context c
         File deviceInfoFile = new File(directorySubfolder, ErrorLogHelper.DEVICE_INFO_FILE);
```

Comment:
I think we'd better use another filename(not `ErrorLogHelper.DEVICE_INFO_FILE,` but something new) here so as to distinct deviceInfoFile from previous versions.

Reply:
And also I believe we should have some migration handling.

Reply:
The logic for parsing old version was added in a0aee3db93d4baaec4fd99300f81991876730410